### PR TITLE
Add Endless OS

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -16,10 +16,12 @@ If you have problems when running Podman in [rootless](README.md#rootless) mode
 sudo yum -y install podman
 ```
 
-#### [Fedora-CoreOS](https://coreos.fedoraproject.org), [Fedora SilverBlue](https://silverblue.fedoraproject.org)
+#### [Fedora-CoreOS](https://coreos.fedoraproject.org), [Fedora SilverBlue](https://silverblue.fedoraproject.org), [Endless OS](https://endlessos.com)
```
Comment (vrothberg) [inline install.md]: Please add an extra entry for Endless OS. openSUSE Kubic has the has a dedicated one as well.

**Comment by vrothberg** [comment]
@LeandroStanger, to unblock the CI, the commit needs to be signed. You can do that via `git commit -s --amed` and then pushing again.

**Comment by LeandroStanger** [comment]
```
Hi @LeandroStanger, thanks for the PR. Just to double check: Podman is part of the default installation of Endless OS? So if I install it, I can run it without the need to install?
```
Yes

**Comment by LeandroStanger** [comment]
git commit -s --amed

**Comment by vrothberg** [comment]
@LeandroStanger, the first commit is still not signed. You can do a `git rebase -i HEAD~~` and squash the two together. Make sure to have the `Signed-off-by: ...` line.

**Comment by LeandroStanger** [comment]
git rebase -i HEAD~~

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Comment by vrothberg** [comment]
> @LeandroStanger, the first commit is still not signed. You can do a `git rebase -i HEAD~~` and squash the two together. Make sure to have the `Signed-off-by: ...` line.

@LeandroStanger, the issue is still there. Commit https://github.com/containers/libpod/pull/3396/commits/45d902b71b316ca16bb6387eb54ce32f86516d7e needs the `Signed-off-by: ...` line.

**Comment by LeandroStanger** [comment]
My system is all bug. I'll close this PR.

**Pull request closed without merging**